framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -66,3 +47,768 @@ info:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /delegates:
+    get:
+      tags:
+        - Delegates
+      summary: Requests delegates data.
+      description: Search for a specified delegate in the system.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+      responses:
+        200:
+          description: Registered delegate accounts.
+          schema:
+            $ref: '#/definitions/DelegateResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /forgers:
+    get:
+      tags:
+        - Delegates
+      summary: Requests next forgers data.
+      description: |
+        Returns a list of the next forgers in this delegate round.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: A list of the next forgers.
+          schema:
+            $ref: '#/definitions/ForgersResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /blocks/{id}:
+    get:
+      tags:
+        - Blocks
+      summary: Requests block data
+      description: Search for block in the system by ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          description: id of the searching block in base64 format
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Block requested.
+          schema:
+            $ref: '#/definitions/BlockResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        404:
+          description: Data not found.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /blocks:
```

Comment:
height parameter is missing?

Reply:
`height` is query since params is already used by `id`